lib/plug/static.ex

```diff
@@ -77,6 +77,11 @@ defmodule Plug.Static do
 
     * `:headers` - other headers to be set when serving static assets.
 
+    * `:types` - custom MIME type mapping. As a map with filename as key
+      and content type as value. For example,
+      `types: %{ "apple-app-site-association" => "application/json" }`.
```

Comment:
Map should not have spaces after `%{` and before `}`.
Also let's use `:` instead of `,`:
```
For example: `%{"apple-app-site-association" => "application/json"}`.
```